tunnel/intercept/svcpoll.go

```diff
@@ -105,47 +109,65 @@ func diffServices(edgeServices []edge.Service, knownServices map[string]edge.Ser
 	return
 }
 
-func updateServices(context ziti.Context, interceptor Interceptor, resolver dns.Resolver, added, removed, all map[string]edge.Service) {
+func updateServices(context ziti.Context, interceptor Interceptor, resolver dns.Resolver, added, removed, all map[string]*entities.Service) {
 	log := pfxlog.Logger()
 	for _, svc := range added {
 		if stringz.Contains(svc.Permissions, "Dial") {
```

Comment:
We don't fail here if no config is found because proxy works fine without configuration